sdk/translation/ai-translation-text/README.md

```diff
@@ -0,0 +1,343 @@
+if (transliterateResponse.status !== "200") {
+  const error = transliterateResponse.body as ErrorResponseOutput;
+  throw error.error;
+}
+
+if (isUnexpected(transliterateResponse)) {
+  throw transliterateResponse.body;
+}
+
+const translations = transliterateResponse.body as TransliteratedTextOutput[];
+for (const transliteration of translations) {
+  console.log(
+    `Input text was transliterated to '${transliteration?.script}' script. Transliterated text: '${transliteration?.text}'.`
+  );
+}
+```
+
+Please refer to the service documentation for a conceptual discussion of [transliterate][transliterate_doc].
+
+### Break Sentence
+
+Identifies the positioning of sentence boundaries in a piece of text.
+
+```typescript
+const inputText: InputTextItem[] = [{ text: "zhè shì gè cè shì。" }];
+const parameters: FindSentenceBoundariesQueryParamProperties & Record<string, unknown> = {
+  language: "zh-Hans",
+  script: "Latn",
+};
+const breakSentenceResponse = await translationClient.path("/breaksentence").post({
+  body: inputText,
+  queryParameters: parameters,
+});
+
+if (breakSentenceResponse.status !== "200") {
+  const error = breakSentenceResponse.body as ErrorResponseOutput;
+  throw error.error;
+}
+
+if (isUnexpected(breakSentenceResponse)) {
+  throw breakSentenceResponse.body;
+}
+
+const breakSentences = breakSentenceResponse.body as BreakSentenceItemOutput[];
+for (const breakSentence of breakSentences) {
+  console.log(`The detected sentece boundaries: '${breakSentence?.sentLen.join(", ")}'.`);
+}
+```
+
+Please refer to the service documentation for a conceptual discussion of [break sentence][breaksentence_doc].
+
+### Dictionary Lookup
+
+Returns equivalent words for the source term in the target language.
+
+```typescript
+const inputText: InputTextItem[] = [{ text: "fly" }];
+const parameters: LookupDictionaryEntriesQueryParamProperties & Record<string, unknown> = {
+  to: "es",
+  from: "en",
+};
+const dictionaryResponse = await translationClient.path("/dictionary/lookup").post({
+  body: inputText,
+  queryParameters: parameters,
+});
+
+if (dictionaryResponse.status !== "200") {
+  const error = dictionaryResponse.body as ErrorResponseOutput;
+  throw error.error;
+}
+
+if (isUnexpected(dictionaryResponse)) {
+  throw dictionaryResponse.body;
+}
+
+const dictionaryEntries = dictionaryResponse.body as DictionaryLookupItemOutput[];
+for (const dictionaryEntry of dictionaryEntries) {
+  console.log(
+    `For the given input ${dictionaryEntry?.translations?.length} entries were found in the dictionary.`
+  );
+  console.log(
+    `First entry: '${dictionaryEntry?.translations[0]?.displayTarget}', confidence: ${dictionaryEntry?.translations[0]?.confidence}.`
+  );
+}
+```
+
+Please refer to the service documentation for a conceptual discussion of [dictionary lookup][dictionarylookup_doc].
+
+### Dictionary Examples
+
+Returns grammatical structure and context examples for the source term and target term pair.
+
+```typescript
```

Comment:
We typically prefer to have a simple starter experience, and offer code snippets in JS instead of TS so it can work for a broader audience. Would you mind if we convert the snippets here to use JS syntax and no types?

Reply:
Converted to JS